src/renderer/styles/android.css

```
@@ -0,0 +1,1316 @@
+		width: 26px;
+		height: 26px;
+		min-width: 26px;
+		min-height: 26px;
+		margin-right: 7px;
+		background: transparent;
+	}
+
+	html.platform-android .subnav-badge,
+	html.mobile-phone .subnav-badge {
+		background: rgba(103, 80, 164, 0.14);
+		color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)